auth/http.go

```diff
@@ -114,6 +114,22 @@ type userResponseJson struct {
 	InternalUUID  string
 }
 
+type userGroups struct {
```

Comment:
First comment that I would like to make - your code does not build.

I imagine that for the moment, it is because you have not updated the go.mod file - and that authaus must be merged, tagged and re-released before this gets merged or something like that

Reply:
yes, wont build without authaus